time stamp
string menu_data
int32 menu_cnt_data
int32 total
